metagraph/tests/annotation/row_diff/test_row_diff.cpp

```diff
@@ -187,47 +187,59 @@ TEST(RowDiff, GetAnnotationMasked) {
     graph.mask_dummy_kmers(1, false);
 
     // build annotation
-    sdsl::bit_vector bterminal = { 0, 0, 0, 0, 1, 0, 1, 0 };
+    sdsl::bit_vector bterminal_masked = { 0, 0, 0, 0, 1, 0, 1, 0 };
+    sdsl::bit_vector bterminal(graph.max_index() + 1);
+    sdsl::bit_vector cols_masked[2] = {
+        { 1, 0, 0, 0, 0, 0, 0, 0 },
+        { 0, 0, 0, 0, 1, 0, 1, 1 }
+    };
+    sdsl::bit_vector cols_concrete[2];
+    cols_concrete[0].resize(graph.max_index() + 1);
+    cols_concrete[1].resize(graph.max_index() + 1);
+    graph.call_nodes([&](auto i) {
+        auto rank = graph.rank_node(i) - 1;
+        bterminal[i - 1] = bterminal_masked[rank];
+        cols_concrete[0][i - 1] = cols_masked[0][rank];
+        cols_concrete[1][i - 1] = cols_masked[1][rank];
```

Comment:
should we replace these `-1`s with `AnnotatedDBG::graph_to_anno_index`?

Reply:
Done